src/test/java/net/sf/jabref/model/entry/AuthorListTest.java

```diff
@@ -375,6 +375,37 @@ public void testGetAuthor() {
         Assert.assertEquals("von Neumann", author.getLastOnly());
         Assert.assertEquals("Neumann, Jr, J.", author.getNameForAlphabetization());
         Assert.assertEquals("von", author.getVon());
+
+    }
+
+    @Test
+    public void testCompanyAuthor() {
+        Author author = AuthorList.parse("{JabRef Developers}").getAuthor(0);
```

Comment:
I prefer tests in the form
````
Author expected = new Author("Firstname", "F.", null, "Bailey-Jones", null);
Assert.assertEquals(new AuthorList(expected), AuthorList.parse("Firstname Bailey-Jones"));
````
They are shorter and provide better feedback what went wrong parsing the author (instead of `getFirst` was something but expected null, you get a complete diff of all information in the parsed author vs expected author).